app/assets/stylesheets/views/profile.scss

```diff
@@ -101,10 +101,11 @@
   .crayons-logo {
     width: var(--su-9);
     height: var(--su-9);
-    border: var(--su-1) solid var(--profile-brand-color);
+    background-color: var(--profile-brand-color);
+    padding: var(--su-1);
 
     @media (min-width: $breakpoint-m) {
-      border-width: var(--su-2);
+      padding: var(--su-2);
```

Comment:
...orrrrr we could do this (instead of what I suggested in previous comment):

```
.crayons-logo {
  ...
  padding: var(--padding, var(--su-1));
  border-radius: calc(var(--radius) + var(--padding));
  @media (...) {
    --padding: var(--su-2);
  }
}
```